src/NodaTime.Demo/PeriodDemo.cs

```diff
@@ -177,5 +177,28 @@ public void Between_LocalDateTimesWithGivenUnits()
             Assert.AreEqual(23, period.Hours);
             Assert.AreEqual("P2Y264DT23H", period.ToString());
         }
+
+        [Test]
+        public void Normalize()
+        {
+            var original = new PeriodBuilder { Weeks = 2, Days = 5 }.Build();
+
+            var actual = Snippet.For(original.Normalize());
+
+            Assert.AreEqual(
```

Comment:
Can we unwrap this to a single line, and possibly remove the blank lines before? Just thinking about how it'll look in the docs.

Reply:
Sure, pushed now